.flake8.ini

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 120
```

Comment:
.travis.yml now says 85 instead of 120 https://github.com/fossasia/query-server/blob/master/.travis.yml#L12

Reply:
@cclauss I will fix this 👍